pipeline/scripts/ci/manage_reports.py

```diff
@@ -0,0 +1,137 @@
+"""This script is to update report artifacts on IBM
+
+- Upload test result files to "qe-ci-reports" bucket
+- Executes bucket file options like upload, download, delete and list
+"""
+
+import logging
+import sys
+
+from docopt import docopt
+
+from storage.ibm_cos import CloudObjectStorage
+
+LOG = logging.getLogger(__name__)
+
+usage = """
+This script helps to update reports in IBM.
+
+Usage:
+  manage_reports.py (upload|download) <bucket_name> (-f FILE_PATH) (-k OBJ_KEYNAME)
+  manage_reports.py delete_object <bucket_name> (-k OBJ_KEYNAME)
+  manage_reports.py list_objects <bucket_name>
```

Comment:
example talks about `delete` instead of `delete_object` ... can we leave it is just `delete`